yt_dlp/extractor/mediasite.py

```diff
@@ -162,9 +162,8 @@ def __extract_slides(self, *, stream_id, snum, Stream, duration, images):
             'url': slide_base_url,
             'protocol': 'mhtml',
             'acodec': 'none',
-            'vcodec': 'jpeg',
-            'quality': -12,
-            'format_note': 'Slides',
+            'vcodec': 'none',
+            'format_note': 'Slides (jpeg)',
```

Comment:
Strictly speaking, `jpeg` is not completely true here anyway, as I mentioned before.

Reply:
is there a way to extract the format? If not, the format note should perhaps just be `Slides`. Having no info is better than wrong info

Reply:
Well, one can request individual images and check the `Content-type` header, or sniff the magic number, like the downloader does. The real issue, though, is what to do when there are multiple formats in use. The Wiles lecture already has the default placeholder slide image in PNG, where the normal slides are JPEGs.

So if you’re going that route, better leave it out entirely.